index.html

```diff
@@ -134,18 +137,8 @@ <h4>Tonic</h4>
             </div>
           </div>
         </div>
-      </section>
 
-      <section class="work-card">
         <div class="project-container">
```

Comment:
Great job @fullstop125 🤝

- [ ] You may please add an image to the second card. This will guide your project to correctly mimic the Figma design.
- Your design
![empty card](https://user-images.githubusercontent.com/63917725/178040352-48d3b363-9765-419b-91f0-24096a64000a.PNG)

Reply:
resolved